Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -125,6 +126,8 @@ class TextureCacheBase
   virtual void DeleteShaders() = 0;
 
   TCacheEntry* Load(const u32 stage);
+  static void InvalidateAllBindPoints() { invalidated_binds = 0xff; }
```

Comment:
Actually, thinking about it more, if you went this route, it might be cleaner to change this to "available_bind_points", then this just becomes:

```
available_bind_points.reset()
```